### ISSUE TITLE
'Undefined function or variable rjtipF_mex' when running Manip_v6_2

### ISSUE THREAD

**Issue by EthanTang0115** [smhadisadati/TMTDyn#2]
Hello developer,

After downloading from your repo, I tried running Manip_v6_2.m, when it enters animation section, the following errors come:

Undefined function or variable 'rjtipF_mex'.

The 'code' folder is included, but I can find rjtipF_mex.mexa64, but matlab doesn't recognize this as a function. 

Is there any configuration, or code modification need to be done before running the code ?

**Comment by smhadisadati** [comment]
Hi Ethan. This is a matlab mex file and machine specific, meaning that you need to derive the model at least once on your own machine (computer) for it to work properly.
Please set the "par.derive=1;" and wait for it to derive the hole model once.

If you need to change any parameter in the subsequent runs, you need to comment the section "%% derivation simplification" that replaces the symbolic variables with constant values. This way, it takes longer to derive the EOM but you do not need to rederive the EOM every time you change a single parameter.

**Comment by EthanTang0115** [comment]
Thanks I will have a try.

**Comment by EthanTang0115** [comment]
A new issue comes up. Actually same as Issue#1. Do you have any comments on that ?
